### ISSUE TITLE
Setting HELP_URL blank doesn't work as documented in .env in 2.5.6

### ISSUE THREAD

**Issue by iangilmour** [bigbluebutton/greenlight#1568]
Just updated to Greenlight 2.5.6.
In my .env I set HELP_URL to blank. On Greenlight restart the help URL remains accessible to the user. Commenting it out completely also has no effect.

Below is the relevant .env section (note: I also have REPORT_ISSUE_URL set to blank - this does work as documented):
```
# The link to the Report an Issue button that appears on the 500 page and in the Account Dropdown
#
# Defaults to the Github Issues Page for Greenlight
# Button can be disabled by setting the value to blank
#
# REPORT_ISSUE_URL=https://github.com/bigbluebutton/greenlight/issues/new
REPORT_ISSUE_URL=

# The link to the Need help? button that appears on the Account Dropdown
#
# Defaults to the Greenlight documentation
# Button can be disabled by setting the value to blank
# HELP_URL=https://docs.bigbluebutton.org/greenlight/gl-overview.html
HELP_URL=
```

**Comment by farhatahmad** [comment]
It was set to default to the our docs for backwards compatibility. We'd probably want to make the change so it works as expected (Default if ENV variable is not there, Hide if ENV variable is blank, Display link if ENV is set)

**Comment by farhatahmad** [comment]
Fixed in v2.6
